src/cartesian/CartesianGrid.tsx

```diff
@@ -21,8 +21,8 @@ interface InternalCartesianGridProps {
   y?: number;
   width?: number;
   height?: number;
-  horizontalCoordinatesGenerator?: (props: any) => number[];
-  verticalCoordinatesGenerator?: (props: any) => number[];
+  horizontalCoordinatesGenerator?: (props: any, syncWithTicks: Boolean) => number[];
+  verticalCoordinatesGenerator?: (props: any, syncWithTicks: Boolean) => number[];
```

Comment:
```suggestion
  verticalCoordinatesGenerator?: (props: any, syncWithTicks: boolean) => number[];
```

Reply:
https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript

Reply:
+1